.travis.yml

```diff
@@ -1,12 +1,29 @@
 language: cpp
 compiler:
   - gcc
+addons:
+  apt:
+    sources:
+      - george-edison55-precise-backports # cmake 3.2.3 / doxygen 1.8.3
```

Comment:
What is this in more detail?

Reply:
We use cpprestsdk for a platform agnostic rest API to send events to event hub and download models from storage.  (https://github.com/Microsoft/cpprestsdk).  This needs a newer version of CMake.

Reply:
This is needed for Ubuntu 14.4.  Later versions of Ubuntu have the newer version of CMake.